devtools/chain/config.toml

```diff
@@ -1,5 +1,6 @@
 # crypto
```

Comment:
### Suggestions
- add friendly comments for the test keys (net_privkey and bls_privkey) in the config file.
- add WARNING: DO NOT USE this private key in any production environment!

Reply:
I agree with that the private key should be saved as a separate file. This can refactor later.